util/chplenv/chpl_llvm.py

```diff
@@ -72,6 +72,8 @@ def get():
         elif (compatible_platform_for_llvm_default()):
             if has_compatible_installed_llvm():
                 llvm_val = 'system'
+        elif (chpl_platform.get('target' == 'linux32')):
```

Comment:
I think this can just be `else` - right? Along with maybe renaming `compatible_platform_for_llvm_default` to `compatible_platform_for_llvm` ?

Reply:
It seems to me we should actually generate an error for linux32 if `CHPL_LLVM` is explicitly set, too.